src/toil_scripts/adam_pipeline/spark_toil_script.py

```diff
@@ -74,9 +74,21 @@ def call_conductor(masterIP, inputs, src, dst):
                 "-e", "AWS_SECRET_KEY="+inputs['secretKey'],
                 "quay.io/ucsc_cgl/conductor",
                 "--master", "spark://"+masterIP+":"+SPARK_MASTER_PORT,
+                "--conf", "spark.driver.memory=%s" % inputs["driverMemory"],
+                "--conf", "spark.executor.memory=%s" % inputs["executorMemory"],
                 "--", "-C", src, dst])
 
 
+def remove_file(masterIP, filename):
+    """
+    Remove the given file from hdfs with master at the given IP address
+    """
+    containerID = check_output(["ssh", "-o", "StrictHostKeyChecking=no", masterIP, "docker", "ps", \
+                                "|", "grep", "apache-hadoop-master", "|", "awk", "'{print $1}'"])[:-1]
```

Comment:
For small outputs it may be easier and more readable to do the filtering and column extraction in Python rather than piping through grep and awk.